### PR TITLE
fix CatBoostRanker score computation

### PR DESCRIPTION
**Pull request by dmitrySorokin**
fix for https://github.com/catboost/catboost/issues/2230

### CODE REVIEW

File: catboost/python-package/catboost/core.py [hunk 1]
```diff
@@ -6228,7 +6228,7 @@ def get_ndcg_metric_name(values, names):
             raise CatBoostError("group_id must be initialized. If groups are not expected, pass an array of zeros")
 
         predictions = self.predict(X)
-        return _eval_metric_util([y], [predictions], get_ndcg_metric_name(), None, group_id, group_weight, None, None, thread_count)[0]
+        return _eval_metric_util([y], [predictions], get_ndcg_metric_name([top, type, denominator], ['top', 'type', 'denominator']), None, group_id, group_weight, None, None, thread_count)[0]
```
[the remark this answers is not in the document]
Comment (dmitrySorokin) [reply catboost/python-package/catboost/core.py:6231]: Done

**Review by Evgueni-Petrov-aka-espetrov — APPROVED** [review]

**Comment by Evgueni-Petrov-aka-espetrov** [comment]
@dmitrySorokin could you please read and sign the Yandex CLA so that we could merge this PR?
https://catboost.ai/en/docs/concepts/development-and-contributions#yandex-cla

**Comment by dmitrySorokin** [comment]
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=ru

**Comment by arcadia-devtools** [comment]
CLA already signed

**Comment by Evgueni-Petrov-aka-espetrov** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
@Evgueni-Petrov-aka-espetrov, internal review request created: [3181666](https://a.yandex-team.ru/review/3181666)

**Comment by Evgueni-Petrov-aka-espetrov** [comment]
@dmitrySorokin 
the internal PR has been merged, and will appear on github soon
many thanks for fixing this bug 🔥 🔥 🔥

**Comment by andrey-khropov** [comment]
This PR has been merged in https://github.com/catboost/catboost/commit/ea09fef7a704dce8d28a5e6ad2a8eed60e3ede6d

**Pull request closed without merging**